### ISSUE TITLE
I got perplexity 0 in this code

### ISSUE THREAD

**Issue by jiho980224** [pytorch/examples#730]
hi, I cloned your code and just ran it. But It shows perplexity 0 all the time..
Did I do something wrong?
and I changed learning rate to 0.31. (But the perplexity was 0 before and after this as well)

It seems like loss is going to explode.
Can somebody help me?
![image](https://user-images.githubusercontent.com/61861533/76369896-7518b400-6378-11ea-8f45-ce4d2ba6098a.png)
